src/main/java/net/imagej/ops/image/distancetransform/DefaultDistanceTransform.java

```diff
@@ -0,0 +1,347 @@
+package net.imagej.ops.image.distancetransform;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+
+import net.imagej.ops.Contingent;
+import net.imagej.ops.Ops;
+import net.imagej.ops.create.img.CreateImgFromDimsAndType;
+import net.imagej.ops.special.function.Functions;
+import net.imagej.ops.special.function.UnaryFunctionOp;
+import net.imagej.ops.special.hybrid.AbstractUnaryHybridCF;
+import net.imglib2.FinalInterval;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.type.BooleanType;
+import net.imglib2.type.numeric.RealType;
+import net.imglib2.type.numeric.real.FloatType;
+
+import org.scijava.plugin.Parameter;
+import org.scijava.plugin.Plugin;
+import org.scijava.thread.ThreadService;
+
+/**
+ * @author Simon Schmid (University of Konstanz)
+ */
+@Plugin(type = Ops.Image.DistanceTransform.class)
```

Comment:
if you set the `priority` property in the `@Plugin` annotation, you don't have to use the `numDims > 3` check in the conforms. Just give this slow implementation a very low priority.